lib/sablon/template.rb

```diff
@@ -32,6 +33,10 @@ def render(context, properties = {})
             out.write(process(Processor::Document, content, env))
           elsif entry_name == 'word/numbering.xml'
             out.write(process(Processor::Numbering, content))
+          elsif entry_name == 'word/_rels/document.xml.rels'
+            out.write(process(Processor::Image, content, properties))
```

Comment:
Yeah I realize that is a non-trival suggestion. I may throw some code your way later on depending on how late I stay in the office since I had a few ideas as I was typing up the comment.